src/main/java/org/reaktivity/command/log/internal/LoggableRoutes.java

```diff
@@ -149,6 +151,19 @@ private String extension(
             hostname != null ? String.format("\"%s\"", hostname) : null,
             applicationProtocol != null ? String.format("\"%s\"", applicationProtocol) : null);
         }
+        else if ("http2".equals(nukleusName))
+        {
+            HttpRouteExFW exFW = new HttpRouteExFW();
+            final int index = route.extension().offset();
+            exFW.wrap(route.extension().buffer(), index, index + route.extension().sizeof());
+            HttpHeaderFW authority = exFW.headers().matchFirst(h -> h.name().asString().equals(":authority"));
+            extension = String.format(
+                    "{" +
+                    "\"headers\":{\"%s\": \"%s\"}" +
+                    "}",
+                     authority.name().asString(),
+                     authority.value().asString());
+        }
```

Comment:
Please show all the headers in the `http2` route extension. Also, avoid using `FW` at end of flyweight variable names.